release_notes.md

```diff
@@ -6,6 +6,7 @@
 - Configure host.json to use workflow when creating a default host.json and app is identified as a logic app. (#6810)
 - Update Node.js Worker Version to [2.1.0](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v2.1.0)
 - Update Java Worker Version to [1.8.1](https://github.com/Azure/azure-functions-java-worker/releases/tag/1.8.1)
+- Update PowerShell Worker to [3.0.629](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.629) and [3.0.630](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.630)
```

Comment:
very minor suggestion: specify the PS version next to each version for clarity

Reply:
Thanks, done.